.tool-versions

```diff
@@ -1,2 +1 @@
-erlang 23.2
```

Comment:
would you mind reverting this back for now?  Our local dev is still mostly elixir 1.11

Reply:
The elixir 1.12/OTP 24 migration is still pending for us, there were a few breaking changes in some dependencies that were a huge pain to figure out but we def want to do it.

Reply:
Absolutely, it was a bit too much to add this into this unrelated PR :D